README.md

```diff
@@ -1,16 +1,35 @@
-# Strapi application
+# Setting up Strapi
 
-Setting up Strapi
+### Quickstart
 
-The first thing you do after forking the ACC-repo is to run "npm run build" and "npm run develop" to start the server, but as you will see, its not that easy.
+###### This section only works for mac and linux
 
-To get started with Strapi you need to install PostgreSQL, and for that you need Homebrew. HomeBrew is a package manager that takes care of everything NPM or Yarn doesn´t. Read more about it on https://brew.sh/
+Windows sections will be added later. For now check out these links to set up:
 
-This is the website that will get you started. Most of the steps here are taken directly from there. https://www.moncefbelyamani.com/how-to-install-postgresql-on-a-mac-with-homebrew-and-lunchy/ PS: This is only for mac. Not sure if it works the same way with Linux or Windows.
+- Windows: https://www.microfocus.com/documentation/idol/IDOL_12_0/MediaServer/Guides/html/English/Content/Getting_Started/Configure/_TRN_Set_up_PostgreSQL.htm
 
-To install HomeBrew, run this command in your terminal:
+#### Proceedings for Mac users
```

Comment:
This is to general. Instead of "Proceedings" say something like "Setting up database"